tests/test_keyboard_shortcuts_route.py

```diff
@@ -0,0 +1,14 @@
+OK_STATUS_CODE = 200
+DATA_IN_SHORTCUTS_PAGE = b"General Shortcuts"
```

Comment:
No difference between the URL and this. Put both in the same scope.

Reply:
Thanks